independent-projects/tools/devtools-testing/src/main/java/io/quarkus/devtools/testing/codestarts/QuarkusCodestartTest.java

```diff
@@ -215,9 +215,7 @@ private Path getProjectWithMockedDataDir(Language language) throws IOException {
 
     @Override
     public void afterAll(ExtensionContext extensionContext) throws Exception {
-        if (enableRegistryClient) {
```

Comment:
Make sure it is disabled anyway (so it can be enabled from outside), this shouldn't have any consequences right @aloubyansky?

Reply:
Disabling a registry client w/o this check may reset a registry client config that wasn't originally enabled by the test. What is the reason for this change?

Reply:
It's in `afterAll` so it should be safe anyway no?

I added this because in `standaloneExtensionCatalog()`, the test registry needs to be enabled before we load the catalog (else if fails). I could also move that standalone catalog logic to the beforeAll..

Reply:
This doesn't make sense to me so far. If it didn't enable the config in the test setup it should not be disabling it at the end.